### DISCUSSION TITLE
Calendar Holidays for only certain language/location

### DISCUSSION THREAD

**Discussion by Chalkie1983** [Intermesh/groupoffice#934] [Q&A] [answered]
Hi there, 

I am new to GroupOffice however I seem to be having one issue, I only want the public holidays from the /language/holidays/en_uk.php and /language/holidays/en.php files, I deleted every other one and then did the holiday cache clear and the normal cache clear however its still loading them all. 

Am I missing something?

Thanks

**Comment by mschering** [comment, marked as answer]
Hi,

I can't replicate the behaviour you describe. If I select "English / United Kingdom" for my user I get the right holidays in the calendar:

![image](https://user-images.githubusercontent.com/1314447/206992615-e5645f28-de7e-4f73-aa75-699b7ca63e78.png)


Best regards,
Merijn

**Reply by Chalkie1983** [reply]
Merijn you absolute star that was what i was missing, the look at feel setting for holidays. 

As I say I am new to this system but it ticks all the boxes I need.

Kind regards,

Adam

**Comment by mschering** [comment]
Good to hear!
